.github/workflows/test.yml

```diff
@@ -123,3 +123,6 @@ jobs:
           echo "This is a schedule event"
           poetry run pytest -vv tests/
           poetry run pytest --openai_model=gpt-4o -m chat -vv tests/test_openai.py
+
+      - name: Test cli commands
```

Comment:
one last comment, how about we separate the llm test and cli tests. So in the Github CI part, we would see which pass and which fail?